server/datastore/mysql/hosts.go

```diff
@@ -360,6 +377,15 @@ func (ds *Datastore) DeleteHost(ctx context.Context, hid uint) error {
 			return ctxerr.Wrapf(ctx, err, "deleting pack_targets for host %d", hid)
 		}
 
+		// no point trying the uuid-based tables if the host's uuid is missing
+		if hostUUID != "" {
+			for table, col := range additionalHostRefsByUUID {
+				if _, err := tx.ExecContext(ctx, fmt.Sprintf(`DELETE FROM %s WHERE %s=?`, table, col), hostUUID); err != nil {
```

Comment:
should we do? not critical because this is not user input.

```suggestion
				if _, err := tx.ExecContext(ctx, fmt.Sprintf("DELETE FROM `%s` WHERE `%s`=?", table, col), hostUUID); err != nil {
```

Reply:
Yeah the test would catch it, but still, might as well do it now should it be needed!